workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/src/plugin.ts

```diff
@@ -14,7 +14,6 @@
  * limitations under the License.
  */
 import '@patternfly/react-core/dist/styles/base-no-reset.css';
-import '@patternfly/patternfly/patternfly.css';
```

Comment:
I think we should have this import as well like tekton, `import '@patternfly/patternfly/utilities/Accessibility/accessibility.css';`

Reply:
Added: https://github.com/backstage/community-plugins/pull/3827/commits/f753c4c568f1314209139cd6874bf78533f31bba